internal/lib/sandbox/sandbox.go

```diff
@@ -502,3 +513,84 @@ func (s *Sandbox) Ready(takeLock bool) bool {
 
 	return cState.Status == oci.ContainerStateRunning
 }
+
+func (s *Sandbox) SetResolvPath(resolvPath string) {
```

Comment:
wouldn't these setters be able to be justl ocal to `builder.go` because it's all the same package?